app/src/main/java/com/nuvoex/baseframework/MainActivity.java

```diff
@@ -37,4 +56,22 @@ public String getName() {
     public boolean onNavigationItemSelected(MenuItem item) {
         return false;
     }
+
+    @Override
+    public void onPermissionGranted(int requestCode) {
+        Log.i("TAG", "onPermissionGranted : " + requestCode);
+
+    }
+
+    @Override
+    public void deniedPermissionAccess(int requestCode) {
```

Comment:
Can you unify these event names? example: onPermissionDenied, onPermissionRemoved etc. Will make it easier to remember them.